lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -49,7 +49,12 @@ public static function getObjectFieldValue($object, $field)
         $accessors = ['get', 'is'];
 
         foreach ($accessors as $accessor) {
-            $accessor .= $field;
+            // Don't duplicate the accessor when the field already starts with it
+            if (0 === strpos($field, $accessor)) {
```

Comment:
I'm afraid this check on its own will lead to false positives: `issue`, `island`, `isolation`. Better heuristic would be to look for `is` followed by a capital letter

Reply:
But then it can be worse performace-wise, not sure if it's worth the hassle